src/Common/src/Interop/Windows/kernel32/Interop.EncryptDecrypt.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class Kernel32
+    {
+        /// <summary>
+        /// WARNING: This method does not implicitly handle long paths. Use EncryptFile.
+        /// </summary>
+        [DllImport(Libraries.Advapi32, EntryPoint = "EncryptFileW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
+        private static extern bool EncryptFilePrivate(string path);
```

Comment:
nit: we are not consistent in this regard but @JeremyKuhne Jeremy and I talked about using the same name as in the docs or header of the native apis. In this case lpPathName